src/main/kotlin/com/ampnet/headlesscmsservice/service/impl/TranslationServiceImpl.kt

```diff
@@ -0,0 +1,31 @@
+package com.ampnet.headlesscmsservice.service.impl
+
+import com.ampnet.headlesscmsservice.enums.Lang
+import com.ampnet.headlesscmsservice.exception.ErrorCode
+import com.ampnet.headlesscmsservice.exception.InternalException
+import com.ampnet.headlesscmsservice.service.TranslationService
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import mu.KLogging
+import org.springframework.stereotype.Service
+
+@Service
+class TranslationServiceImpl(
+    private val objectMapper: ObjectMapper
+) : TranslationService {
+
+    companion object : KLogging()
+
+    private val translations by lazy {
+        val json = javaClass.classLoader.getResource("mail_translations.json")?.readText()
+            ?: throw InternalException(ErrorCode.INT_CMS_DEFAULT_MAIL, "Could not find mail_translations.json")
+        objectMapper.readValue<Map<String, Map<String, String>>>(json)
+    }
+
+    override fun getTranslation(key: String, lang: Lang): String =
+        translations[key]?.get(lang.name.toLowerCase()) ?: translations[key]?.get(Lang.EN.name.toLowerCase())
```

Comment:
Will we always use lower case? Can we override `Lang.name` to always return lower case?